openapi/nytimes.books/types.bal

```diff
@@ -15,200 +15,200 @@
 // under the License.
 
 public type InlineResponse2003ResultsBooks record {
-    string age_group?;
-    string author?;
-    string contributor?;
-    string contributor_note?;
-    string created_date?;
-    string description?;
-    int price?;
-    string primary_isbn10?;
-    string primary_isbn13?;
-    string publisher?;
-    int rank?;
-    string title?;
-    string updated_date?;
+    string? age_group?;
```

Comment:
Fields are not originally nillable. Please regenerate without giving `--nullable` in the command.